recipes/default.rb

```diff
@@ -15,51 +15,58 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-include_recipe 'osl-repos::centos'
-include_recipe 'osl-repos::epel'
-include_recipe 'osl-mysql::client'
-include_recipe 'base::python'
 
-# WARNING!
-# If this gets updated, all NodeJS apps running will need to have their
-# node_modules directories completely removed and `npm install` run again to
-# update the modules to match the new Node version's ABI.
-node.override['nodejs']['repo'] = 'https://rpm.nodesource.com/pub_6.x/el/$releasever/$basearch'
+if node['platform_version'].to_i <= 7
```

Comment:
I think this is better.
```suggestion
if node['platform_version'].to_i < 8
```